sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/EventTimeWatermark.scala

```diff
@@ -40,7 +41,8 @@ object EventTimeWatermark {
 case class EventTimeWatermark(
     eventTime: Attribute,
     delay: CalendarInterval,
-    child: LogicalPlan) extends UnaryNode {
+    child: LogicalPlan) extends UnaryNode
+    with Logging {
```

Comment:
Are we using this anywhere ?

Reply:
I had added it for some logging information. Removed it.